.travis.yml

```diff
@@ -18,11 +18,15 @@ php:
   - 5.4
   - 5.5
   - 5.6
-
+  - 7.0
+    
 matrix:
   fast_finish: true
+  allow_failures:
+    - php: 7.0
 
 before_install:
+  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then sudo cp tests/php7-pool.conf ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.d/; fi
```

Comment:
Yes, so I heard from @julienbourdeau 
But then I came across this blogpost: https://www.marcus-povey.co.uk/2016/02/16/travisci-with-php-7-on-apache-php-fpm/